src/main/java/seedu/typed/model/ModelManager.java

```diff
@@ -149,8 +149,9 @@ public String toString() {
         @Override
         public boolean run(ReadOnlyTask task) {
             return nameKeyWords.stream()
-                    .filter(keyword -> StringUtil.containsWordIgnoreCase(task.getName().getValue(), keyword)).findAny()
-                    .isPresent();
+                    .filter(keyword -> StringUtil
+                    .containsFuzzyWordIgnoreCase(task.getName().getValue(), keyword))
```

Comment:
In particular, the problem is FindCommandParser is supposed to parse a string and return a command but made no use of the other methods in the body.